Clinical-Genomics/chanjo2
# Load a demo case when demo app is launched

### This PR adds | fixes:
- Loads a demo case when demo app is launched (fix #32)

### How to test:
- Launch demo app
- Use cases endpoint to make sure demo case is there

### Review:

This [version](https://semver.org/) is a:
